plugin/evm/config.go

```diff
@@ -90,9 +94,13 @@ type Config struct {
 	KeystoreInsecureUnlockAllowed bool   `json:"keystore-insecure-unlock-allowed"`
 
 	// Gossip Settings
-	RemoteTxGossipOnlyEnabled bool     `json:"remote-tx-gossip-only-enabled"`
-	TxRegossipFrequency       Duration `json:"tx-regossip-frequency"`
-	TxRegossipMaxSize         int      `json:"tx-regossip-max-size"`
+	RemoteGossipOnlyEnabled       bool             `json:"remote-gossip-only-enabled"`
+	RegossipFrequency             Duration         `json:"regossip-frequency"`
+	RegossipMaxTxs                int              `json:"regossip-max-size"`
+	PriorityRegossipFrequency     Duration         `json:"tx-priority-regossip-frequency"`
+	PriorityRegossipMaxTxs        int              `json:"tx-priority-regossip-max-size"`
+	PriorityRegossipAddresses     []common.Address `json:"tx-priority-regossip-addresses"`
+	PriorityRegossipTxsPerAddress int              `json:"priority-regossip-txs-per-address"`
```

Comment:
should likely match `tx-priority-regossip-address-txs` per the docs

Reply:
whoops!